phoenix-client-parent/pom.xml

```diff
@@ -48,7 +48,15 @@
           <skip>true</skip>
         </configuration>
       </plugin>
-
+      <plugin>
+        <!-- RAT gets really confused on the phoenix-client subprojects, 
+        and there is nothing to audit here -->
+        <groupId>org.apache.rat</groupId>
+        <artifactId>apache-rat-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
And `phoenix-client` is going to run RAT check even in the presence of this right? It overrides plugin from parent pom?

Reply:
No it won't. We could re-enable RAT by specifying skip=false in the submodules, but there is nothing to audit in either submodule, they both have only a single pom.xml file in them.

Reply:
Oh yes, there is nothing in sub modules either.